api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -1272,7 +1272,9 @@ public static void stopModule(Module mod, boolean isShuttingDown) {
 	}
 
 	/**
-     * Merged collapsible if statement
+	 * Replaces further nested if statement with && on the outer if statement instead
+	 * @param dependentModule dependent module to check
+	 * @param modulePackage if the dependent module contains this String
```

Comment:
The above boolean param is not well described. Just check at the existing javadoc for boolean params for an example.